install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
```

Comment:
I actually intentionally wanted retry attempts to overwrite previous attempts rather than litter /var/tmp with files that would never be deleted, since it's annoyingly hard/fragile to make a bash script reliably clean up after itself in error cases. I guess the multiple user point is sort of plausible, in the sense of "it will probably never happen but it could theoretically happen". I suppose I could throw $USER into the filenames?